src/com/content-box/box.js

```diff
@@ -50,11 +50,7 @@ export default class ContentBox extends Component {
 
 	render( props, state ) {
 		props = Object.assign({}, props);
-
-		var node = props.node;
-		var user = props.user;
-		var path = props.path;
-		var extra = props.extra;
+		const {node, user, path, extra} = props;
```

Comment:
Hmm! I didn't realize this was working with the compiler. You might be on to something here. Collapsing many `var = props` lines in to one, I think I like it. :smile: